platform/platform_tests/distance_tests.cpp

```diff
@@ -250,17 +252,17 @@ UNIT_TEST(Distance_FormattedDistance)
     {Distance(991,     Units::Meters),     990,  Units::Meters,     "990",  "990 m"},
```

Comment:
Could you also add all new tests from this PR: https://github.com/organicmaps/organicmaps/pull/5505?